.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,89 @@
+name: Bug
+description: File a bug report
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a bug report, please search for the behaviour in the existing issues. If this is regarding NS functionality, please report to NS customer Support Serivce.
+        
+        ---
+        
+        Thank you for taking the time to file a bug report. To address this bug as fast as possible, we need some information.
+  - type: dropdown
+    id: tool
+    attributes:
+      label: Select the SuiteCloud tool where you encountered the issue
+      options:
+        - SuiteCloud CLI
+        - SuiteCloud Extension for VSCode
+        - SuiteCloud Unit Testing
```

Comment:
Is it possible to remove the none option from the options? It appears in drop down if I "preview" this file.